python/tvm/topi/cuda/batch_matmul.py

```diff
@@ -258,11 +260,20 @@ def batch_matmul_int8(cfg, x, y, out_shape=None, out_dtype=None):
     out_dtype : Optional[str]
         Specifies the output data type for mixed precision batch matmul.
 
+    transpose_a : Optional[bool] = False
+        Whether the first tensor is in transposed format.
+
+    transpose_b : Optional[bool] = True
+        Whether the second tensor is in transposed format.
+
     Returns
     -------
     output : tvm.te.Tensor
         3-D with shape [batch, M, N]
     """
+    del out_shape
```

Comment:
Is this because this compute doesn't process this? What happen if we just leave it as an unused argument? Or we could check if `out_shape` is consistent with the inferred output shape and throw a warning if not.

Reply:
Yeah, just some linting tools (like ctypes) may throw an error on unused parameters. Since we've not enabled the ctypes check here, leave this here will not affect anything.